cloudquery-cloud/manifest.json

```
@@ -0,0 +1,35 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "727e53a9-0bbe-4277-8ed0-9e9425fe34de",
+  "app_id": "cloudquery-sync",
```

```suggestion
  "app_id": "cloudquery-cloud",
```
To match the offering name. This informs the URL structure.